sections/main-product.liquid

```diff
@@ -71,12 +71,25 @@
             {%- assign media = host_product.featured_media -%}
             <li class="product__media-item grid__item slider__slide{% if media.media_type != 'image' %} product__media-item--full{% endif %}{% if section.settings.hide_variants and variant_images contains media.src %} product__media-item--variant{% endif %}" data-media-id="{{ section.id }}-{{ media.id }}">
               {% render 'product-thumbnail', media: media, position: 'featured', loop: section.settings.enable_video_looping, modal_id: section.id, xr_button: true %}
+              {%- if section.settings.show_image_label -%}
+              <div class="image_label">
+                <div id="image_type">Stock Photo</div>
```

Comment:
id-s generally used very sparingly and only they are strictly necessary (input labels and anchors are 2 common cases), because it's easy to create conflicts because there must ever only be 1 of each id used on the entire page and id's essentially form a global namespace.
I'm not sure there is a good reason to use an id to refer to image_type instead of a class

Reply:
Changed to class, I´ll stick away from the id´s.